package.json

```diff
@@ -37,7 +38,7 @@
   },
   "devDependencies": {
     "check-ecmascript-version-compatibility": "^0.1.1",
-    "csv-parse": "^4.4.5",
+    "csv-parse": "^4.4.6",
```

Comment:
Why was this update needed?

Reply:
it was part of npm audit fix for https://npmjs.com/advisories/1171, i will make this fix as separate PR

Reply:
I'll update all of the devDependencies separately—no need to make a separate PR.